common/changes/@gooddata/sdk-ui-all/pb-unknown-visclass-fallback_2023-04-05-10-36.json

```diff
@@ -0,0 +1,10 @@
+{
+    "changes": [
+        {
+            "packageName": "@gooddata/sdk-ui-all",
+            "comment": "Add fallback for unknown visualization classes",
```

Comment:
I will reword this to "Add fallback for unknown visualization type" classes sound like technical detail

Reply:
Updated according to the discussion.